plugins/meta/portmap/main.go

```diff
@@ -164,7 +164,10 @@ func parseConfig(stdin []byte, ifName string) (*PortMapConf, error) {
 			}
 
 			// Skip known non-sandbox interfaces
-			intIdx := ip.Interface
+			if ip.Interface == nil {
```

Comment:
I'm not sure this is right - the interface is nil when up-converting v0.2.0 results, right?